packages/forms-web-app/src/views/manage-appeals/appeal-details.njk

```diff
@@ -79,6 +79,14 @@
               value: {
                 text: appeal.LPAApplicationReference
               }
+            },
+            {
+              key: {
+                text: "Application form"
+              },
+              value: {
+                text: ["<a href='", applicationForm.documentURI, "'>", applicationForm.filename, "</a>" ] | join | safe
+              }
```

Comment:
I believe this should sit in a separate table, its currently in the appeal headlines rather than the submission table, which is added in a PR i have open.

Though we can always move this once that's merged if easier.

Reply:
The appeal headlines table is not in this PR. The way the tickets are sliced makes keeping track of things rather challenging. The table you see is the submission table.